controllers/users_test.go

```diff
@@ -10,7 +11,12 @@ import (
 	"github.com/google/uuid"
 )
 
-func createUserTestData() (*test.OrderedTests, error) {
+const (
+	CreateUserTest = 0
+	DeleteUserTest = 1
```

Comment:
```suggestion
	CreateUserTest = iota
	DeleteUserTest
```
For more information see [here](https://yourbasic.org/golang/iota/).